homework-3/src/test/java/RandomString.java

```diff
@@ -0,0 +1,31 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomString {
+    public static void main(String[] args) {
+        String[] stringsList = {"Hello World", "Have a great day", "Angel", "Random String", "Word Game"};
+        List<String> filteredStringList = new ArrayList<String>();
+
+
+        for(Integer i = 0; i < stringsList.length; i++) {
+            String item = stringsList[i];
+            Integer itemLength = item.length();
```

Comment:
it's better to use primitive type everywhere where we don't need to use Objects
int > Integer in this case